ManuProjectFiles/app/src/main/java/com/example/manu/QuizActivity.kt

```diff
@@ -191,9 +191,10 @@ class QuizActivity : AppCompatActivity() {
     }
 
     /**
-     * Loads the menu screen.
+     * Loads the "Return to Menu?" popup screen.
```

Comment:
I don't feel like you need a question, its like you're asking if its loading the return to menu screen.